### ISSUE TITLE
Resource property description doesn't show up in the editor

### ISSUE THREAD

**Issue by MeroVinggen** [godotengine/godot#96728]
### Tested versions

- Reproducible in: 4.3-stable, 4.2.2-stable

### System information

Godot v4.2.2.stable - Windows 10.0.22631 - Vulkan (Mobile) - dedicated NVIDIA GeForce GTX 1070 (NVIDIA; 31.0.15.4633) - Intel(R) Core(TM) i7-7700HQ CPU @ 2.80GHz (8 Threads)

### Issue description

The resource exported props with doc. description (via "##") may not show in the editor on mouse hover and print default text instead: "No description available".

P.S.

My actual Windows version is 11, no idea why system shows version as 10.x

### Steps to reproduce

> I can't tell so far the 100% way to reproduce the issue, cause 3 times I tried it didn't occur in the same way

0. create new empty project
1. create new script, inherit it from Resource and add 2 exported properties. give the second one a document description

like this:
```
extends Resource
class_name ResDesc

@export var prop2 = 10

## awesome description
@export var prop: int = 1
```

2. create two new resources of it ("ResDesc" in my example)
3. reload the project twice 
4. open the resource in the editor and check the description appears

> Note! As I mentioned above - sometimes it works like it should, but eventually the problem shows up when I create new resources of the same type and added new property above existed one with a description AFTER creation the first resource and few reloads

> At one of the reproducing tries, an error appears at "output" on another project reload. It shows only once but the problem with the missing description remains. Error text: "scene/gui/text_edit.cpp:5286 - Index p_line = -1 is out of bounds (text.size() = 2)."

### Minimal reproduction project (MRP)

N/A

**Comment by dalexeev** [comment]
Most likely a duplicate of #72406 / #86577. Please test if #95821 fixes this.

Thanks for reporting the bug nonetheless!
